SPEC.md

```diff
@@ -65,21 +65,21 @@ Examples: `minitest`, `mysql2`, `battery-horse`.
 
 The version number of the library (String, required).
 
-- It should follow the [Semantic Versioning](http://semver.org/) format.
+- It should follow the [Semantic Versioning 2.0](http://semver.org/spec/v2.0.0.html) format.
```

Comment:
We don't enforce semver (see #10 about that). This is merely a recommendation. The user may choose whatever scheme they want; e.g. [CalVer](http://sedimental.org/calver.html) is a great alternative.